sdk/keyvault/azure-security-keyvault-keys/CMakeLists.txt

```diff
@@ -94,6 +94,15 @@ target_include_directories(
       $<INSTALL_INTERFACE:include>
 )
 
+# Include shared source code
+# NOTE: Use shared-code only within .cpp files. DO NEVER consume the shared-code from header files.
+target_include_directories(
+  azure-security-keyvault-keys
+    PRIVATE
+      $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/sdk/keyvault/azure-security-keyvault-shared/inc>
+      $<INSTALL_INTERFACE:include>
```

Comment:
Ditto "Why `INSTALL_INTERFACE`?" (I'll stop pointing this out despite that I expect more occurences.)